networkx/generators/tests/test_line.py

```diff
@@ -174,6 +174,13 @@ def test_claw(self):
         G.add_edges_from(G_edges)
         pytest.raises(nx.NetworkXError, nx.inverse_line_graph, G)
 
+    def test_edgeless_graph(self):
+        # This graph has 2 nodes and 0 edges
+        G = nx.Graph()
+        G_nodes = [0, 1]
+        G.add_nodes_from(G_nodes)
```

Comment:
Just a nit, but this could be streamlined a bit

```suggestion
        G = nx.Graph()
        G.add_nodes_from([0, 1])
```